egs/librimix/ConvTasNet/local/conf.yml

```diff
@@ -0,0 +1,37 @@
+encoder:
+  enc_name : free
+decoder:
+  dec_name : free
+
+# filterbank config
+filterbank:
+  n_filters: 512
+  kernel_size: 16
+  stride: 8
+# Network config
+masknet:
+  n_blocks: 6
+  n_repeats: 2
+  mask_act: relu
+  bn_chan: 128
+  skip_chan: 128
+  hid_chan: 512
+# Training config
+training:
+  epochs: 200
+  batch_size: 4
+  num_workers: 2
+  half_lr: yes
+  early_stop: yes
+# Optim config
+optim:
+  optimizer: adam
+  lr: 0.001
+  weight_decay: 0.
+# Data config
+data:
+  metadata_train_path : "/home/jcosentino/jcosentino/libri/small_snr/8K/min/metadata/mixture_train-clean-360.csv"
+  metadata_val_path : "/home/jcosentino/jcosentino/libri/small_snr/8K/min/metadata/mixture_dev-clean.csv"
```

Comment:
We need a local path here, not an absolute one.
So the csv files need to be stored in data/wav8k/min/tr for example

Reply:
Sure, but can you explain why please ?

Reply:
The user will modify one thing before running the recipe: `storage_dir` in run.sh. 
After that, the .csv files are stored in ./data and this path is given to the training script. 
This is one less thing for the user to do, so it is better.
If not stored, maybe symlinked.